src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_params.py

```diff
@@ -46,6 +46,18 @@
 register_cli_argument('resource feature list', 'resource_provider_namespace', CliArgumentType(options_list=('--namespace',), required=False, help=_PROVIDER_HELP_TEXT))
 register_cli_argument('resource feature', 'feature_name', CliArgumentType(options_list=('--name', '-n'), help='the feature name'))
 
+register_cli_argument('resource policy', 'resource_group_name', arg_type=resource_group_name_type, completer=get_resource_group_completion_list)
+register_cli_argument('resource policy', 'policy_definition_name', options_list=('--name', '-n'), help='name of policy definition')
+register_cli_argument('resource policy create', 'rules', help='JSON formatted string or a path to a file with such content')
```

Comment:
>JSON formatted string [](start = 63, length = 21)

I'm still not sold on the JSON formatted string as an input, but I won't hold up the PR over it. I'll run it by Johan when he gets back. He used to be against it but since the generic update uses JSON he might be fine with it.